sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -57,8 +57,8 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     (result, expected) match {
       case (result: Array[Byte], expected: Array[Byte]) =>
         java.util.Arrays.equals(result, expected)
-      case (result: Double, expected: Spread[Double]) =>
-        expected.isWithin(result)
+      case (result: Double, expected: Spread[_]) => // Can't use Spread[Double] b/c of erasure
```

Comment:
I see. Sadly, I think this is not going to work here without extra work, and then it is not going to do what you want. This version of scalatest uses manifest to encode type information, and you would have to define it manually in this context:
```scala
    implicit val x: Manifest[Int] = ???
    stream shouldBe a [ReceiverInputDStream[Int @unchecked]]
```
but then the scalatest library is not aware of the `unchecked` annotation, and still throws a warning. Let's just have `_` in the suite file.